UsabillaSampleApp/README.md

```diff
@@ -26,4 +35,124 @@ $ cordova-res
 $ cordova-res ios
 ```
 
-Otherwise, if `config.xml` exists, `cordova-res` will look for platforms (e.g. `<platform name="ios">`) and generate resources only for the configured platforms.
\ No newline at end of file
+Otherwise, if `config.xml` exists, `cordova-res` will look for platforms (e.g. `<platform name="ios">`) and generate resources only for the configured platforms.
+
+### Add Platform
+
+- To add native platforms like android & ios , just run:
+
+```bash
+$ cordova platform add android ios
+```
+
+### Add Usabilla plugin
+
+- To add usabilla into cordova application , just run:
```

Comment:
To add the Usabilla SDK into your Cordova application, just run:

Reply:
Done